### ISSUE TITLE
"Stochastic ...'s value is outside its support" with 2.3.3

### ISSUE THREAD

**Issue by elpres** [pymc-devs/pymc3#643]
A model of mine works fine with 2.3.2, but after upgrading to 2.3.3 it started failing with the error in the title. The affected variables are not the same at every run, but they are all of the same kind, used for standard deviations and created like this:

```
sig_... = pm.Exponential('sig_...', beta=0.1, value=1)
```

(I chose a rather flat exponential distribution because of its shape, positive and with high density in the lower end, and not because I think it's the best choice of a prior distribution in this case.) If I look at their traces with 2.3.2, they look reasonable, with a maximum at, e.g. 3.0, falling off sharply to the left and smoothly to the right. With 2.3.3 however, sometimes the trace looks normally distributed around 0, sometimes it is the correct distribution but mirrored around the y-axis and below 0 as a whole.

I would like to provide any information required to debug the problem, but the model is a rather complex one, and if I remove some variables, the problem occurs less frequently, and some runs succeed without errors. So minimizing it to a trivial example might be challenging. Is there anything else in terms of diagnostics that I could extract?

**Comment by fonnesbeck** [comment]
Can you install from the 2.3 branch on GitHub? If you are using pip, you can do this via:

```
pip install -U git+git://github.com/pymc-devs/pymc.git@2.3
```

This will install 2.3.4. See if you can replicate this there. I think I know what the issue is.

**Comment by elpres** [comment]
You are right, the problem seems to be solved in the current 2.3 branch. I'll just wait until there is an updated version in the repo.  Thanks for your help!
